### PR TITLE
Bug 1127170 - Reduce the bottom padding for shadow canvas to make no overlap between icon and text, r=kgrandon

### PR DESCRIPTION
**Pull request by howareyou322**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6db9d6713b11d1fd9f6ddf19bb2a264853cf7e77

File: shared/elements/gaia_grid/js/grid_icon_renderer.js [hunk 1]
```diff
@@ -43,6 +48,18 @@
     _createCanvas: function() {
       var canvas = document.createElement('canvas');
       canvas.width = this._maxSize + (CANVAS_PADDING * 2);
+      canvas.height = this._maxSize + CANVAS_PADDING + SHADOWCANVAS_PADDING_BOTTOM;
```
Comment (KevinGrandon) [inline shared/elements/gaia_grid/js/grid_icon_renderer.js]: Jshint is complaining about this error saying it's too long.

File: shared/elements/gaia_grid/js/grid_icon_renderer.js [hunk 2]
```diff
@@ -11,6 +11,11 @@
   const DEFAULT_BACKGROUND_COLOR = 'rgb(228, 234, 238)';
   const UNSCALED_CANVAS_PADDING = 2;
   const CANVAS_PADDING = UNSCALED_CANVAS_PADDING * devicePixelRatio;
+  /**
+   * Only append one pixel padding in the bottom for shadow canvas to
+   * get a round circle
+   */
+  const SHADOWCANVAS_PADDING_BOTTOM = 1 * devicePixelRatio;
```
Comment (KevinGrandon) [inline shared/elements/gaia_grid/js/grid_icon_renderer.js]: Please make this ```SHADOW_CANVAS_PADDING_BOTTOM``` for readability.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=93bc8b68a692508f5018fe7723e96594a4e067b0

**Comment by autolander** [comment]
The pull request could not be integrated in as it contained a commit which was missing a bug number. Please ensure all commits contain bug numbers, and are in the format of: Bug [number] - [description]

**Comment by autolander** [comment]
The pull request could not be integrated in as it contained a commit which was missing a bug number. Please ensure all commits contain bug numbers, and are in the format of: Bug [number] - [description]

**Pull request merged**